examples/crud-gorm/handlers/handler.go

```diff
@@ -0,0 +1,75 @@
+package handlers
+
+import (
+	"strconv"
+
+	"github.com/go-fuego/fuego"
+	"github.com/go-fuego/fuego/examples/crud-gorm/models"
+	"github.com/go-fuego/fuego/examples/crud-gorm/queries"
+)
+
+type Handlers struct {
+	UserQueries *queries.UserQueries
+}
+
+type UserToCreate struct {
+	Name  string `json:"name"`
+	Email string `json:"email"`
+}
+
+func (h *Handlers) GetUsers(c fuego.ContextNoBody) ([]models.User, error) {
+	return h.UserQueries.GetUsers()
+}
+
+func (h *Handlers) GetUserByID(c fuego.ContextNoBody) (models.User, error) {
+	id, err := strconv.Atoi(c.PathParam("id"))
+	if err != nil {
+		return models.User{}, fuego.BadRequestError{}
+	}
+	user, err := h.UserQueries.GetUserByID(uint(id))
+	if err != nil {
+		return models.User{}, err
+	}
+	return *user, nil
+}
+
+func (h *Handlers) UpdateUser(c fuego.ContextWithBody[models.User]) (models.User, error) {
+	id, err := strconv.Atoi(c.PathParam("id"))
+	if err != nil {
+		return models.User{}, fuego.BadRequestError{}
+	}
+	input, err := c.Body()
+	if err != nil {
+		return models.User{}, fuego.BadRequestError{}
+	}
+	input.ID = uint(id)
+	err = h.UserQueries.UpdateUser(&input)
+	if err != nil {
+		return models.User{}, err
+	}
+	return input, nil
```

Comment:
It feels odd that you're returning he input. I believe you'd want to return the updated User. I realize that these should be the same, but it seems odd. It would also get rid of a few lines of code.